udata/core/issues/api.py

```diff
@@ -110,20 +85,46 @@ def post(self, id):
             on_issue_closed.send(issue, message=message)
         else:
             on_new_issue_comment.send(issue, message=message)
-        return marshal(issue, issue_fields), 200
+        return issue
 
 
 @ns.route('/', endpoint='all_issues')
-class ListIssuesAPI(API):
+@api.doc(parser=parser)
+class IssuesAPI(API):
     '''
     List all issues.
     '''
-    @api.doc(id='list_issues', parser=parser)
+    @api.doc('list_issues')
     @api.marshal_with(issue_page_fields)
     def get(self):
-        '''List all Issues for an object given its ID'''
+        '''List all Issues'''
         args = parser.parse_args()
         issues = Issue.objects
+        if args['for']:
+            issues = issues(subject__in=args['for'])
         if not args['closed']:
             issues = issues(closed=None)
         return issues.paginate(args['page'], args['page_size'])
+
+    @api.secure
+    @api.doc(model=issue_fields)
```

Comment:
Redundant with `@api.marshal_with`. Can be removed.
Be it lacks the input model (`@api.expect`)